AxonDeepSeg/models/download_model.py

```diff
@@ -0,0 +1,17 @@
+from AxonDeepSeg.ads_utils import download_data
+import shutil
+
+def main():
+    url_TEM_model = "https://osf.io/2hcfv/?action=download&version=4"  # URL of TEM model hosted on OSF storage with the appropriate versioning on OSF
+    url_SEM_model = "https://osf.io/sv7u2/?action=download&version=4"  # URL of SEM model hosted on OSF storage with the appropriate versioning on OSF
+
+    if (not download_data(url_TEM_model) and not download_data(url_SEM_model)) == 1:
+        print('Data downloaded and unzipped succesfully.')
+    else:
+        print('ERROR: Data was not succesfully downloaded and unzipped- please check your link and filename and try again.')
+    shutil.move("default_SEM_model", "AxonDeepSeg/models/default_SEM_model")
```

Comment:
Not sure if this command will work for Windows; maybe we should use a Path() object instead of hardcoding the path. Also, if called from a different folder than the root project, it might not place the data in the correct location. I'll open an issue, this is a minor issue but in need of a quick fix nonetheless.

Reply:
Opened #286